From 0d82ede25b01d625f404089d6b5f8136051a9e67 Mon Sep 17 00:00:00 2001
From: Anish Trivedi <anish@freescale.com>
Date: Fri, 20 Nov 2009 15:04:19 -0600
Subject: [PATCH 136/505] ENGR00118659 MX51: SCC2 test fails due to incorrect
 ioctl code definitions

Removed ioctl macro redefinitions so that original ioctl macros generate unique codes

Signed-off-by: Anish Trivedi <anish@freescale.com>
---
 include/scc2_test_driver.h | 13 -------------
 include/scc_test_driver.h  | 13 -------------
 2 files changed, 26 deletions(-)

diff --git a/include/scc2_test_driver.h b/include/scc2_test_driver.h
index cf08d51..6acb649 100644
--- a/include/scc2_test_driver.h
+++ b/include/scc2_test_driver.h
@@ -79,19 +79,6 @@
 /** @{ */
 /** ioctl cmd to return version and configuration information on driver and
  *  SCC2 */
-#ifdef _IOW
-#undef _IOW
-#endif
-#ifdef _IOWR
-#undef _IOWR
-#endif
-#ifdef _IO
-#undef _IO
-#endif
-
-#define _IOW(x, y, z) y
-#define _IOWR(x, y, z) y
-#define _IO(x, y) y
 
 #define SCC2_TEST_GET_CONFIGURATION _IOW (SCC2_TEST_DRIVER_IOCTL_IDENTIFIER, 0, \
          scc_configuration_access)
diff --git a/include/scc_test_driver.h b/include/scc_test_driver.h
index a97871d..b08fa41 100644
--- a/include/scc_test_driver.h
+++ b/include/scc_test_driver.h
@@ -111,19 +111,6 @@ extern int scc_test_major_node;
 /** @{ */
 /** ioctl cmd to return version and configuration information on driver and
  *  SCC */
-#ifdef _IOW
-#undef _IOW
-#endif
-#ifdef _IOWR
-#undef _IOWR
-#endif
-#ifdef _IO
-#undef _IO
-#endif
-
-#define _IOW(x, y, z) y
-#define _IOWR(x, y, z) y
-#define _IO(x, y) y
 
 #define SCC_TEST_GET_CONFIGURATION _IOW (SCC_TEST_DRIVER_IOCTL_IDENTIFIER, 0, \
          scc_configuration_access)
-- 
1.8.0

